examples/MLIRSparseTensor/makefile

```diff
@@ -44,3 +44,10 @@ sparse-tensor-pack-run:
 		--sparse-compiler="enable-runtime-library=false" | \
 	${MLIR_CPU_RUNNER} -e main -O0 --entry-point-result=void \
 		--shared-libs=${MLIR_RUNNER_UTILS},${MLIR_C_RUNNER_UTILS}
+
+# `sparse_tensor.expand/compress` are lowering operation
```

Comment:
The statement here seems ambiguous. Can we use a statement like "`sparse_tensor.expand/compress` operations are lowered from high-level linalg operation"?